assets/scss/style.scss

```diff
@@ -8,7 +8,6 @@
 	Stable tag: 2.3.1
 	Requires at least: 4.7
 	Tested up to: 5.6
-	Requires PHP: 5.6
```

Comment:
Restore

Reply:
@matipojo I am not sure what to restore? I Removed the "Requires PHP: 5.6" since it was added in the last commit.

Reply:
I have adde back the row: Requires PHP: 5.6